README_NEW.md

```diff
@@ -0,0 +1,654 @@
+
+
+## Aggregator
+
+### Run
+
+To start the aggregator run:
+
+```bash
+make aggregator_start CONFIG_FILE=<path_to_config_file>
+```
+
+If you want to run the aggregator with the default configuration, you can run:
+
+```bash
+make aggregator_start
+```
+
+### Config
+
+There is a default configuration for devnet purposes in `config-files/config.yaml`.
+
+The configuration file has the following structure:
+
+```yaml
+# Common variables for all the services
+# 'production' only prints info and above. 'development' also prints debug
+environment: <production/development>
+aligned_layer_deployment_config_file_path: <path_to_aligned_layer_deployment_config_file>
+eigen_layer_deployment_config_file_path: <path_to_eigen_layer_deployment_config_file>
+eth_rpc_url: <http_rpc_url>
+eth_ws_url: <ws_rpc_url>
+eigen_metrics_ip_port_address: <ip:port>
+
+## ECDSA Configurations
+ecdsa:
+  private_key_store_path: <path_to_ecdsa_private_key_store>
+  private_key_store_password: <ecdsa_private_key_store_password>
+
+## BLS Configurations
+bls:
+  private_key_store_path: <path_to_bls_private_key_store>
+  private_key_store_password: <bls_private_key_store_password>
+
+## Aggregator Configurations
+aggregator:
+  server_ip_port_address: <ip:port>
+  bls_public_key_compendium_address: 
+  avs_service_manager_address: 
+  enable_metrics: <true/false>
+```
+
+
+## Operator
+
+### Register into EigenLayer
+
+To register an operator in EigenLayer run the following command:
+
+```bash
+make operator_register_with_eigen_layer CONFIG_FILE=<path_to_config_file>
+```
+
+To register an operator in EigenLayer with the default configuration, you can run:
+
+```bash
+make operator_register_with_eigen_layer
+```
+
+### Deposit Strategy Tokens
+
+#### Anvil
+
+There is an ERC20 token deployed in the Anvil chain to use as strategy token with EigenLayer.
+
+To deposit strategy tokens in the Anvil chain, you can use the following command:
+
+```bash
+make operator_mint_mock_tokens CONFIG_FILE=<path_to_config_file>
+make operator_deposit_into_mock_strategy CONFIG_FILE=<path_to_config_file>
+```
+
+To deposit strategy tokens in the Anvil chain with the default configuration, you can run:
+
+```bash
+make operator_mint_mock_tokens
+make operator_deposit_into_mock_strategy
+```
+
+#### Holesky/Mainnet
+
+EigenLayer strategies are available in [eigenlayer-strategies](https://holesky.eigenlayer.xyz/restake).
+
+For Holesky, we are using [WETH](https://holesky.eigenlayer.xyz/restake/WETH) as the strategy token.
+
+To obtain HolETH and swap it for different strategies, you can use the following [guide](https://docs.eigenlayer.xyz/eigenlayer/restaking-guides/restaking-user-guide/stage-2-testnet/obtaining-testnet-eth-and-liquid-staking-tokens-lsts).
+
+### Register into AlignedLayer
+
+To register an operator in AlignedLayer run the following command:
```

Comment:
replace AlignedLayer for Aligned

Reply:
Solved in [9e80cac](https://github.com/yetanotherco/aligned_layer/pull/152/commits/9e80cac2c185ef42fd5c9c6439b43a9eab385cee)